src/main/frontend/state.cljs

```diff
@@ -496,7 +496,7 @@ should be done through this fn in order to get global config and config defaults
 (defn get-scheduled-future-days
   []
   (let [days (:scheduled/future-days (get-config))]
-    (or (when (int? days) days) 0)))
+    (or (when (int? days) days) 14)))
```

Comment:
Changing the default may surprise people as it's on the homepage. Should we mention it in the release note somehow? There's a related comment in the default config to update

Reply:
I just updated it back to 7.